Cargo.toml

```diff
@@ -14,6 +14,7 @@ reqwest = "0.6"
 serde = "1"
 serde_json = "1"
 serde_derive = "1"
+url = "1.5"
```

Comment:
`url` was already a dependency of `reqwest`, just exposing it to this crate here.